tests/bin/check_tron_jobs_test.py

```diff
@@ -4,13 +4,32 @@
 import time
 
 import check_tron_jobs
+import mock
 from check_tron_jobs import State
+from mock import PropertyMock
 from testify import assert_equal
 from testify import TestCase
 
 
 class CheckJobsTestCase(TestCase):
 
+    def test_check_job_result_exception(self):
+        with mock.patch('check_tron_jobs.parse_cli') as mock_args, \
+                mock.patch('check_tron_jobs.cmd_utils'), \
+                mock.patch('check_tron_jobs.Client') as mock_client, \
+                mock.patch('check_tron_jobs.check_job_result') as mock_check_job_result:
+
+            type(mock_args.return_value).job = PropertyMock(return_value=None)
```

Comment:
These (lines 22-29) look like assignments, should they be assertions?

Reply:
anddd I completely misread that. Still confused why you need to set something on the type, though.

Reply:
According to the document here:  https://docs.python.org/3/library/unittest.mock.html#unittest.mock.PropertyMock

Since the way mock attributes are stored, a PropertyMock can't be attached to a mock object directly.